server/repository/src/diesel_macros.rs

```diff
@@ -169,54 +154,46 @@ macro_rules! apply_simple_string_filter {
 /// Example expand, when called with:
 ///
 /// ```
-/// apply_simple_string_or_filter!(query, filter.code_or_name, item_dsl::code, item_dsl::name)
+/// apply_simple_string_filter!(query, filter.comment, invoice_dsl::comment)
 /// ```
 ///
 /// ```
-/// if let Some(string_filter) = filter.code_or_name {
+/// if let Some(string_filter) = filter.comment {
 ///     if let Some(value) = equal_filter.equal_to {
-///         query = query.filter(item_dsl::code.eq(value));
-///         query = query.or_filter(item_dsl::name.eq(value));
+///         query = query.filter(invoice_dsl::comment.eq(value));
 ///     }
 ///
 ///     if let Some(value) = equal_filter.like {
-///         query = query.filter(item_dsl::code.like(format!("%{}%", value)));
-///         query = query.or_filter(item_dsl::name.like(format!("%{}%", value)));
+///         query = query.filter(invoice_dsl::comment.like(format!("%{}%", value)));
 ///     }
 /// }
 /// ```
-#[cfg(not(feature = "postgres"))]
-macro_rules! apply_simple_string_or_filter {
-    ($query:ident, $filter_field:expr, $dsl_field_1:expr, $dsl_field_2:expr ) => {{
-        if let Some(string_filter) = $filter_field {
-            if let Some(value) = string_filter.equal_to {
-                $query = $query.filter($dsl_field_1.eq(value.clone()));
-                $query = $query.or_filter($dsl_field_2.eq(value));
-            }
-
-            if let Some(value) = string_filter.like {
-                // in sqlite like is case insensitive (but on only works with ASCII chars)
-                $query = $query.filter($dsl_field_1.like(format!("%{}%", value.clone())));
-                $query = $query.or_filter($dsl_field_2.like(format!("%{}%", value)));
-            }
-        }
+macro_rules! apply_simple_string_filter {
+    ($query:ident, $filter_field:expr, $dsl_field:expr ) => {{
+        $crate::apply_simple_string_filter_method!($query, filter, $filter_field, $dsl_field);
     }};
 }
-#[cfg(feature = "postgres")]
-macro_rules! apply_simple_string_or_filter {
-    ($query:ident, $filter_field:expr, $dsl_field_1:expr, $dsl_field_2:expr ) => {{
-        if let Some(string_filter) = $filter_field {
-            if let Some(value) = string_filter.equal_to {
-                $query = $query.filter($dsl_field_1.eq(value.clone()));
-                $query = $query.or_filter($dsl_field_2.eq(value));
-            }
 
-            if let Some(value) = string_filter.like {
-                // Use case insensitive like
-                $query = $query.filter($dsl_field_1.ilike(format!("%{}%", value.clone())));
-                $query = $query.or_filter($dsl_field_2.ilike(format!("%{}%", value)));
-            }
-        }
+/// Example expand, when called with:
```

Comment:
A comment here would be great (has to be before ands)

Reply:
done

thanks for the link, yes it took me a while to figure it out correctly...